### PR TITLE
Add a methodify formula.

### LINKED ISSUE #14: interest in adding other transforms like es6.properties.shorthand ?
https://github.com/stefanpenner/es6.properties.shorthand/blob/master/index.js

```js
var a = {
  foo: function() { }
};
```

becomes:

```js
var a = {
  foo() { }
};
```

### PR DESCRIPTION
**Pull request by Serabe**
Fix #14

### CODE REVIEW

File: addon/commands/methodify.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var Watson = require('../../index');
+var watson = new Watson();
+
+module.exports = {
+  name: 'watson:methodify',
+  description: 'Convert to methods to new ES6 syntax.',
```
Comment (knownasilya) [inline addon/commands/methodify.js]: Should 'to' there?

File: addon/commands/methodify.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  works: 'insideProject',
+  anonymousOptions: [
+    '<path>'
+  ],
+  run: function(commandOptions, rawArgs) {
+    var paths = rawArgs || ['app', 'tests'];
+    for(var i = 0, len=paths.length; i < len; i++) {
```
Comment (knownasilya) [inline addon/commands/methodify.js]: spacing around the equals.

**Comment by Serabe** [comment]
@knownasilya, fixed. Thank you.

**Comment by marcioj** [comment]
LGTM.

Can you also add a cli test in [cli-test.js](https://github.com/abuiles/ember-watson/blob/master/tests/cli-test.js) ?

**Comment by Serabe** [comment]
marcioj, just did it. Thank you.

File: tests/cli-test.js [hunk 3]
```diff
@@ -46,12 +50,16 @@ Object.keys(commands).forEach(function (commandName) {
     it('default path', function() {
       cli(['node', 'ember-watson', commandName]);
 
+      console.log("CALLED: ", called);
+      console.log("Command ", command);
```
Comment (marcioj) [inline tests/cli-test.js]: Are these `console.log`s for debug purposes? If so could you remove them?

File: tests/cli-test.js [hunk 4]
```diff
@@ -46,12 +50,16 @@ Object.keys(commands).forEach(function (commandName) {
...
       assert.deepEqual(called, command);
     });
 
     it('custom path', function() {
       cli(['node', 'ember-watson', commandName, 'my/path']);
 
+      console.log("CALLED: ", called);
+      console.log("Command ", command);
```
Comment (marcioj) [inline tests/cli-test.js]: same here

**Comment by Serabe** [comment]
Sorry. Done.

**Comment by abuiles** [comment]
@Serabe thanks! I'll be releasing tonight :)

**Pull request merged**